src/app/containers/FrontPageMain/index.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import FrontPageMain from '.';
+import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
+import { articleDataNews, articleDataPersian } from '../Article/fixtureData';
```

Comment:
This should probably be front page fixture data rather than article data, should resolve the comment about the `<h1>` snapshot as well